GlucoseKit/GlucoseStore.swift

```diff
@@ -124,7 +124,7 @@ public final class GlucoseStore: HealthKitSampleStore {
                     let sortedGlucose = glucose.sorted { $0.startDate < $1.startDate }
 
                     self.sampleDataCache.append(contentsOf: sortedGlucose)
-                    self.purgeOldGlucoseSamples()
+                    //self.purgeOldGlucoseSamples()
```

Comment:
Going to close this.  A different approach, that allows dexcom users to continue having dexcom manage its HK data, would be acceptable.